src/Hook/Feature.php

```diff
@@ -141,12 +199,75 @@ public function actionFeatureSave(array $params)
                 'INSERT INTO ' . _DB_PREFIX_ . 'layered_indexable_feature_lang_value
                 (`id_feature`, `id_lang`, `url_name`, `meta_title`)
                 VALUES (
-                ' . (int) $params['id_feature'] . ', ' . (int) $language['id_lang'] . ',
+                ' . $featureId . ', ' . (int) $language['id_lang'] . ',
                 \'' . pSQL(Tools::link_rewrite($seoUrl)) . '\',
                 \'' . pSQL($metaTitle, true) . '\')'
             );
         }
 
         $this->module->invalidateLayeredFilterBlockCache();
     }
+
+    /**
+     * Saves feature form. @since ps 1.7.7
+     *
+     * @param array $params
+     */
+    private function save(array $params)
+    {
+        $featureId = (int) $params['id'];
+
+        $this->cleanLayeredIndexableTables($featureId);
+
+        $this->database->execute(
+            'INSERT INTO ' . _DB_PREFIX_ . 'layered_indexable_feature
+            (`id_feature`, `indexable`)
+            VALUES (' . $featureId . ', ' . (int) $params['form_data']['layered_indexable'] . ')'
+        );
+
+        $defaultLangId = (int) Configuration::get('PS_LANG_DEFAULT');
+        foreach (Language::getLanguages(false) as $language) {
+            $langId = (int) $language['id_lang'];
+            $metaTitle = pSQL($params['form_data']['meta_title'][$langId]);
+            $seoUrl = $params['form_data']['url_name'][$langId];
+            $name = $params['form_data']['name'][$langId] ?: $params['form_data']['name'][$defaultLangId];
+
+            if (empty($seoUrl)) {
+                $seoUrl = $name;
+            }
+
+            $tableName = _DB_PREFIX_ . 'layered_indexable_feature_lang_value';
+            $url = pSQL(Tools::link_rewrite($seoUrl));
+
+            $query = sprintf("
+              INSERT INTO $tableName(`id_feature`, `id_lang`, `url_name`, `meta_title`)
```

Comment:
```suggestion
              INSERT INTO %s (`id_feature`, `id_lang`, `url_name`, `meta_title`)
```
Do not use $ for no reason in string